.github/workflows/test-spec.yml

```diff
@@ -63,10 +63,7 @@ jobs:
             tests: plugins/security
             cert: tests/plugins/security/.kirk.pem
             key: tests/plugins/security/.kirk-key.pem
-          - version: 2.20.0
```

Comment:
On a second thought maybe we should leave this one in here in case there's a 2.20? It also shows how to do a `ref`.

Reply:
There is no plan for 2.20 for the time being, 2.19.x will be the last version of 2.x.
I think we are safe to remove it.

Thanks.